packages/types/src/NetworkInterfaces.ts

```diff
@@ -2,12 +2,78 @@ import { IAccount } from './IAccount';
 import { IStorageEngine } from './IStorageEngine';
 import { SecureContextConfig } from './StorageLinkInterfaces';
 
-export enum EnvironmentType {
-    LOCAL = "local",
+export enum Network {  
+	LOCAL = "local",  
 	DEVNET = "devnet",
-    TESTNET = "testnet",
-    MAINNET = "mainnet"
+	BANKSIA = "banksia",  
+	MYRTLE = "myrtle",  
 }
+  
+/**  
+ * @todo Deprecate in favour of `Network`
+ */
+export enum EnvironmentType {  
+	LOCAL = Network.LOCAL,  
+	DEVNET = Network.DEVNET,  
+	BANKSIA = Network.BANKSIA,  
+	MYRTLE = Network.MYRTLE,  
+
+	/**  
+	 * @deprecated use MYRTLE instead  
+	 */
+	MAINNET = Network.MYRTLE,  
+	/**  
+	 * @deprecated use BANKSIA instead  
+	 */  
+	TESTNET = Network.BANKSIA,  
+}
+
+export enum BlockchainAnchor {
+	POLAMOY = "polamoy",
+	POLPOS = "polpos",
+
+	/**  
+	 * @todo Remove once Amoy is deployed
+	 */
+	MUMBAI = "mumbai",
+
+	/**  
+	 * @deprecated use POLPOS instead  
+	 */
+	MAINNET = "polpos",
+
+	/**  
+	 * @deprecated use MUMBAI instead  
+	 */
+	TESTNET = "mumbai",
+	//TESTNET = "polamoy",
+	DEVNET = "mumbai"
+}
+
+export enum BlockchainNetworks {
+	// mainnet
+	//'mainnet' = "0x89",
+	'polpos' = "0x89",
+
+	// testnet
+	//'testnet' = "0x13881",
+	/**
+	 * @todo Remove once Amoy is deployed
+	 */
+	'mumbai' = "0x13881",
+	'devnet' = "0x13881"
+}
+
+export interface NetworkDefinition {  
+	id: string
+	label: string
+	isMainnet: boolean
+	anchoredBlockchain: BlockchainAnchor
+	tokenAddress: string | null
+	storageNodeRegistryAddress: string | null
+	nameRegistryAddress: string | null
```

Comment:
Does the name registry related to the Network or to the DID VDA? Same question for the DID linkage contract.

Reply:
Both relate tothe network

Reply:
Ok.
I don't understand why for the Name registry, it's supposed to be an alias to the DID.